test/types/logger.test-d.ts

```diff
@@ -5,11 +5,15 @@ import * as pino from 'pino'
 
 expectType<FastifyLoggerInstance>(fastify().log)
 
-;['trace', 'debug', 'info', 'warn', 'error', 'fatal'].forEach(logLevel => {
+class Foo {}
+
+['trace', 'debug', 'info', 'warn', 'error', 'fatal'].forEach(logLevel => {
   expectType<FastifyLogFn>(fastify<Server, IncomingMessage, ServerResponse, FastifyLoggerInstance>().log[logLevel as LogLevel])
   expectType<void>(fastify<Server, IncomingMessage, ServerResponse, FastifyLoggerInstance>().log[logLevel as LogLevel](''))
   expectType<void>(fastify<Server, IncomingMessage, ServerResponse, FastifyLoggerInstance>().log[logLevel as LogLevel]({}))
-  expectError(fastify<Server, IncomingMessage, ServerResponse, FastifyLoggerInstance>().log[logLevel as LogLevel](0))
```

Comment:
add this test back but make it `expectType<void>` instead

Reply:
Updated